IBM/carbon-addons-iot-react
# Tablefilter fix

Closes #1330 

**Summary**

Fixes the clear filter issue that was introduced 

**Change List (commits, features, bugs, etc)**

- add key conditionally to filterheaderrow

**Acceptance Test (how to verify the PR)**

- go to ?path=/story/watson-iot-table--stateful-example-with-expansion-maxpages-and-column-resize
- open knobs and change hasFilter from onKeyPress to onBlur
- press clear filter button
- see that all filters are cleared.
